portability-worker/src/main/java/org/dataportabilityproject/worker/WorkerExtensionContext.java

```diff
@@ -38,11 +40,22 @@
           .put(JsonFactory.class, new JacksonFactory())
```

Comment:
Do we remove the JacksonFactory now that we have the type manager?

Reply:
Sure, I'll do that in this PR